packages/grid/src/vaadin-grid-mixin.js

```diff
@@ -269,6 +277,15 @@ export const GridMixin = (superClass) =>
         }),
       ).observe(this.$.table);
 
+      const minHeightObserver = new ResizeObserver(() =>
+        setTimeout(() => {
+          this.__updateMinHeight();
```

Comment:
The `__updateMinHeight` function gets called on every resize. Not sure if it can be a performance issue:

https://github.com/user-attachments/assets/c53748dc-f4f3-4756-b92c-45fbbc13b535

Reply:
It doesn't seem particularly performance intensive, at least compared to other functions that run on resize. And those would only be triggered from certain explicit user interactions, like resizing a dialog or split layout. 

What would be the alternatives? There is `__updateHeaderAndFooter`, but running this on every content update might be worse, and doesn't seem to cover just changing renderer. Theoretically it might be good enough to just calculate a min height once, and if header and footer size later change that isn't super relevant as the general assumption is that no end-user should ever see a grid in this state. But it seems tricky to determine when the initial configuration of a grid has finished.

Reply:
In theory, `__updateMinHeight` would only need to run if the _height_ of the observed element has changed since the previous run. But if the performance impact would be insignificant, it's probably not worth the extra logic.